tests/foreman/cli/test_rhcloud_inventory.py

```diff
@@ -106,6 +106,11 @@ def test_positive_inventory_recommendation_sync():
 
     :CaseLevel: System
     """
+    org, ak = organization_ak_setup
+    cmd = f'organization_id={org.id} foreman-rake rh_cloud_insights:sync'
+
+    result = rhcloud_sat_host.execute(cmd)
+    assert result.status == 0
```

Comment:
Can you add assertion for `Synchronized Insights hosts hits data` in `result.stdout`